app/src/main/java/com/fstyle/androidtrainning/service/ServicePlayMusic.java

```diff
@@ -6,23 +6,29 @@
 import android.media.MediaPlayer;
 import android.net.Uri;
 import android.os.Binder;
+import android.os.Handler;
 import android.os.IBinder;
 import android.util.Log;
-import com.fstyle.androidtrainning.data.local.storage.ExternalData;
 import com.fstyle.androidtrainning.model.Track;
 import com.fstyle.androidtrainning.utils.Constant;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
 
 import static android.content.ContentValues.TAG;
 
 public class ServicePlayMusic extends Service {
 
     private MediaPlayer mMediaPlayer;
-    private Uri mUri;
-    private ExternalData mExternalData = new ExternalData();
     private int id = 0;
     private int mediaLength = 0;
     private IBinder mIBinder = new LocalBinder();
+    private Handler mHandler = new Handler();
+    private List<Track> mTracks = new ArrayList<>();
+    private boolean isShuffle = false;
```

Comment:
WARNING: [AvoidFieldNameMatchingMethodName] Field isShuffle has the same name as a method
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName